### DISCUSSION TITLE
Question: read from replicas timeline

### DISCUSSION THREAD

**Discussion by tsukit** [aembke/fred.rs#50] [General]
@aembke I'd like to check in if you have a better idea of the timeline for supporting reading from replicas as we discussed [here](https://github.com/aembke/fred.rs/issues/32). Thanks!

**Comment by aembke** [comment]
Hey @tsukit,

I haven't started it yet, but it's currently second on the list of new features. I'm in the middle of adding better support for more complicated cluster deployments with TLS, and after that I'll start the replica support. Realistically that probably wont be ready until mid August.

**Reply by tsukit** [reply]
That sounds good. Thank you!
